tests/test_config.py

```diff
@@ -66,6 +66,7 @@ def test_hmac_secret_is_text(self, mocked_render_template):
 
     @mock.patch("kinto.config.render_template")
     def test_init_postgresql_values(self, mocked_render_template):
+        self.maxDiff = None
```

Comment:
why this here?

Reply:
I'll remove it, @Natim added it to debug some test failure that needed some more context.

Reply:
It's not a left over, in case this test fail we will be happy to have it to debug.

Reply:
ok, leaving